notebooks/building_structural_dmg.ipynb

```diff
@@ -0,0 +1,423 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false,
+    "jupyter": {
+     "outputs_hidden": false
+    },
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "from pyincore import IncoreClient, FragilityService, MappingSet\n",
+    "from pyincore.analyses.buildingdamage import BuildingDamage"
```

Comment:
<img width="1205" alt="image" src="https://github.com/IN-CORE/incore-docs/assets/13950475/0be06feb-b994-4741-9ed2-4a165a216089">

Don't you want to use the BuildingStructuralDamage instead in this notebook?

Reply:
@navarroc should we also update notebook as well?

Reply:
Sure, let's update the example notebook since it should be easy. The idea of deprecation is so we don't have to update everything at once, but maybe we should at least update the example to be correct.